### PR TITLE
Ruby 3 rails 6

### PR DESCRIPTION
**Pull request by gburgett**

### CODE REVIEW

File: wcc-contentful/lib/wcc/contentful/event.rb [hunk 1]
```diff
@@ -10,6 +10,8 @@ module WCC::Contentful::Event
   def self.from_raw(raw, context = nil)
     const = Registry.instance.get(raw.dig('sys', 'type'))
 
+    return const.new(raw, **context) unless context.nil?
```
[the remark this answers is not in the document]
Comment (gibsorya) [reply wcc-contentful/lib/wcc/contentful/event.rb:13]: I'll take a look at it! It wasn't my favorite solution either, so if I can try something different and see how it goes

**Review by gibsorya — APPROVED** [review]
Looks good to me! 

@gburgett could also use your approval

**Comment by gburgett** [comment]
Can't approve my own PR haha but 👍

**Comment by gburgett** [comment]
@gibsorya please update the root readme before merging

**Comment by gibsorya** [comment]
lol you right!

I guess I was just wanting to make sure that there weren't any more changes needing to be made. I'll make that README change!

**Pull request merged**